media-libs/ptex/ptex-2.1.28-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Per-Face Texture Mapping for Production Rendering"
+HOMEPAGE="http://ptex.us/"
+SRC_URI="https://github.com/wdas/ptex/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+RDEPEND="sys-libs/zlib"
+DEPEND="${RDEPEND}
+	app-doc/doxygen"
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
this whole ordering needs to be cleaned up:

```
LICENSE="BSD"
SLOT="0"
KEYWORDS="~amd64 ~x86"

RDEPEND="sys-libs/zlib:="
DEPEND="${RDEPEND}
	app-doc/doxygen"
```

don't forget the `:=` on zlib!